docs/GE_Checklist.md

```diff
@@ -6,40 +6,138 @@ to be taken in order to publish a new FIWARE GE.
 Any doubt or request to the FIWARE Foundation shall be made by sending an email
 to [fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org)
 
--   New entry in the [FIWARE Catalogue](https://catalogue-server.fiware.org)
-    under the corresponding Chapter agreed at TSC Level. See also
+### New Generic Enablers
+
+The following mimimal requirements must be met before a new contribution can be
+considered as an Incubated Generic Enabler.
+
+-   All New Generic Enablers **MUST** be presented to the TSC for candidature
+    and accepted into an existing Chapter
+-   The codebase **MUST** be available on GitHub
+-   The GitHub repository **MUST** include an appropriate Open Source License
+-   For functional readiness at a minimum, a new incubated enabler **MUST** be
+    able to be instantiated directly in a running state by any competent
+    developer without the need to refer to other dependent technologies - for
+    example:
+
+    -   Keyrock relies on MySQL, but a working MySQL can be easily instantiated
+        by Docker and minimal data is seeded by Keyrock - the database volume
+        can be persisted. Knowledge of SQL is unnecessary for use.
+    -   An IoT Agent can have a dependency on MongoDB - the volumes are exposed
+        so that the data will be persisted if the Docker container goes down.
+        Alternatively an IoT Agent can be memory based, but all configuration
+        can be set up using a config file or Docker params and then data
+        injected by HTTP requests. An understanding of MongoDB is unnecessary to
+        get it to work.
+    -   Quantum Leap works with Grafana and Crate-DB - appropriate set-up and
+        docker compose are available in the Quick Start. Quantum Leap can be
+        used without in depth knowledge of Crate-DB
+
+-   For all webservice components, a `Dockerfile` **MUST** be available within
+    the code-base
+-   Basic documentation **MUST** exist in Markdown and **MUST** be complete
+    enough for usage of the component:
+    -   A Quick Start guide
+    -   Admin Installation instructions - how to create instances:
+        -   Standalone using configuration files
+        -   Using Docker/Docker-compose and injecting `ENV` variables
+    -   Developer Documentation - how to call the API programmatically
+    -   User Documentation - how to use the GUI
+    -   Apiary file (where necessary)
+-   For a stateful component, the state **MUST** be persistable between
+    instantiations, additional manual set-up **MUST** not be required.
+-   There **MUST** be a commitment that the team behind the product will follow
```

Comment:
how that commitment is expressed?

What happens if the commitment is not finally fulfilled? 

How the incubation process terminates? 

Can the incubation process terminate by rejecting the Enabler? What criteria are used for doing so? 

What criteria are used for transitioning to a full Enabler? Should we ask for some numbers on adoption and community creation around the component?

Reply:
Fixed : 9e99c49 - Text expanded. 

The testing of the commitment  should be part of standard QA testing (as it is for all other enablers)

Failure to comply will eventually result in the same sanction as a full member that does not comply - the Component will be quarantined and eventually removed. The only difference is that an Incubated Enabler has more leeway and time to come up to standard.

Successful progression to full member should be rubber-stamped by the TSC.